exec/cnex/cnex.c

```diff
@@ -76,7 +77,9 @@ BOOL ParseOptions(int argc, char* argv[])
             if(argv[nIndex][1] == 'H' || argv[nIndex][1] == 'h' || argv[nIndex][1] == '?' || ((argv[nIndex][1] == '-' && argv[nIndex][2] != '\0') && (argv[nIndex][2] == 'H' || argv[nIndex][2] == 'h'))) {
                 showUsage();
                 exit(1);
-            } else if (argv[nIndex][1] == 'D' || argv[nIndex][1] == 'd') {
+            } else if (argv[nIndex][1] == 'D') {
```

Comment:
So now some command line options do different things with uppercase and lowercase, and others accept *either* uppercase or lowercase? Be consistent. Choose one style (case-sensitive or not) and stick with it.

Reply:
It's fine to expect command line options that are case sensitive, like `-a` and -A` do different things. It's also fine to expect command line options that are case insensitive, like `-a` and `-A` always do the same thing. But it's not fine to mix *both* styles in the same program. That's the inconsistency.